lib/web_console/session.rb

```diff
@@ -64,6 +64,10 @@ def switch_binding_to(index)
 
     private
 
+      def initial_binding
+        @bindings.find {|b| b.eval('__FILE__').start_with?(Rails.root.to_s) }
```

Comment:
Can we expect that `__FILE__` will always be a string? I think we can hit cases where it is `nil` for bindings in `{instance_,class_,}eval` string blocks, so we probably have to do `b.eval('__FILE__').to_s` in here.

Also a small style nit: `{ |b|` vs `{|b|`.